.github/workflows/build.yml

```diff
@@ -60,6 +60,19 @@ jobs:
       - name: Run Type Check
         run: yarn typecheck
 
+  oxlint:
+    name: oxlint
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Node.js
+        uses: ./.github/actions/setup-node
+        with:
+          electron-install: false
+      - name: Run oxlint
+        run: npx --yes oxlint@latest .
```

Comment:
I don't use yarn, but is there a `yarnx`?

You should put `oxlint` before everything else because this is the fastest to run.